kuma/wiki/tests/test_views_akismet.py

```diff
@@ -132,7 +132,7 @@ def test_spam_no_permission(
     )
     # Redirects to login page when without permission.
     assert response.status_code == 302
-    assert response["Location"].endswith("users/signin?next={}".format(url))
+    assert response["Location"].endswith(f"{reverse(settings.LOGIN_URL)}?next={url}")
```

Comment:
Instead of messing with `settings.MULTI_AUTH_ENABLED` in this unimportant test, how about we just using `settings.LOGIN_URL` correctly?